api/typeDefs/user.js

```diff
@@ -98,6 +98,9 @@ export default gql`
     noteItemMentions: Boolean!
     nsfwMode: Boolean!
     tipDefault: Int!
+    tipRandom: Boolean!
```

Comment:
I named everything `tipRandom...`, following the existing pattern from `tipDefault` and `turboTipping`. Would you prefer me to use `zap` instead? e.g. `zapRandom`, `zapRandomMin`, etc.? @huumn

Reply:
tip is fine. We've been so lazy with the names